osidb/tests/factories.py

```diff
@@ -49,6 +49,15 @@ class Meta:
         yes_declaration=factory.Faker("future_datetime", tzinfo=UTC),
         no_declaration=factory.Faker("past_datetime", tzinfo=UTC),
     )
+    summary = factory.Maybe(
+        "embargoed",
+        yes_declaration=factory.sequence(
```

Comment:
You don't have to use `factory.Sequence` as this field actually depends on `cve_id`. I would instead suggest to use `factory.LazyAttribute` which allows you to reference the generated `cve_id`. Tt is used for `title`, `description` and `statement` for example, you can take the usage idea from there.

Reply:
Actually after reading @osoukup comment the `title` which in fact is a `summary` from SFM2 already uses the `factory.LazyAttribute` so you can just tweak it accordingly.

Reply:
makes sense, changed to reflect the suggestion.